napari/_qt/qt_main_window.py

```diff
@@ -190,6 +192,17 @@ def _add_window_menu(self):
         self.window_menu = self.main_menu.addMenu('&Window')
         self.window_menu.addAction(exit_action)
 
+    def _add_plugins_menu(self):
+        order_plugin_action = QAction("Plugin call order...", self._qt_window)
```

Comment:
The sort plugin window is blocking which makes sense if I think about the code (as are our keybindings / info ones) but it tripped me up a little while testing as I was expecting to be able to toggle boxes on / off or rearrange things and drag and drop at the same time. At first i thought something was wrong with my plugin or file, then i noticed i wasn't seeing any error messages, and then finally i realized that no drag and drop was working at all. It might be nice to have the window be non-blocking to allow people to continue experimenting with drag and drop while the window is open

Reply:
ah good point.  Yeah, if you were confused momentarily that's no good.
I made it modal/blocking just to stay with the general aversion I've sensed here to avoid additional windows being open.  I can definitely see the argument for having it open while trying some io, and I'm fine with that.  Curious whether @jni or @royerloic have other preferences though?  If others _did_ want to keep it modal, we could definitely do more to make it clear that the window needs to be dismissed before proceeding.